TrafficCapture/nettyWireLogging/src/main/java/org/opensearch/migrations/trafficcapture/netty/LoggingHttpRequestHandler.java

```diff
@@ -73,19 +73,7 @@ public LoggingHttpRequestHandler(IChannelConnectionCaptureSerializer trafficOffl
         );
     }
 
-
-
-    protected HttpCaptureSerializerUtil.HttpProcessedState
-    onHttpObjectsDecoded(List<Object> parsedMsgs) throws IOException {
-        parsedMsgs.stream()
-                .filter(o -> o instanceof DefaultHttpRequest)
-                .map(o -> (DefaultHttpRequest) o)
-                .findAny()
-                .ifPresent(req -> getHttpRequestReceiverHandler().currentRequest = req);
-        return HttpCaptureSerializerUtil.addRelevantHttpMessageIndicatorEvents(trafficOffloader, parsedMsgs);
```

Comment:
Don't we still need to call `addRelevantHttpMessageIndicatorEvents` ?

Reply:
Yep - next version does that.  It will include an additional check in a unit test to make sure that the EOM is there for the requests.